docs/user-guide/compute-resources.md

```diff
@@ -49,9 +49,12 @@ Documentation for other releases can be found at
 <!-- END MUNGE: GENERATED_TOC -->
 
 When specifying a [pod](pods.md), you can optionally specify how much CPU and memory (RAM) each
-container needs.  When containers have resource limits, the scheduler is able to make better
-decisions about which nodes to place pods on, and contention for resources can be handled in a
-consistent manner.
+container needs.  When containers have their resource requests and limits specified, the scheduleris
```

Comment:
s/scheduleris/scheduler is/

s/resource requests and limits/resource requests/
(because scheduler does not look at limits)